backend/app/routers/auth.py

```
@@ -46,61 +68,71 @@
-        (User.email == user.email) | (User.username == user.username)).first()
+    existing_user = (
+        db.query(User)
+        .filter((User.email == user.email) | (User.username == user.username))
+        .first()
+    )
 
     if existing_user:
-        raise HTTPException(status_code=400, detail="User with this email or username already exists.")
-    
+        raise HTTPException(
+            status_code=400, detail="User with this email or username already exists."
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)